acme/setup.py

```diff
@@ -76,6 +83,7 @@
     include_package_data=True,
     install_requires=install_requires,
     extras_require={
+        ':python_version<"2.7.9"': py279_extras,
```

Comment:
I have updated by removing the conditional appending to install_requires. Can you try again? (building the wheel from this branch)